homework.md

```diff
@@ -92,22 +93,22 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your code onto a repository on Git
```

Comment:
> Save your code onto a repository on Git
>
So this what you have to understand that **Git** is **tool** that you make use of to save changes to your local and remote repository plus many more other tasks... 

### "Committing is saving all your file changes to your **local repository** by using 'Git' "